launchUI.py

```diff
@@ -1449,9 +1447,9 @@ def create_cgra_pannel(master, rows, columns):
 
     # draw data memory
     memHeight = GRID_HEIGHT
-    button = tkinter.Button(canvas, text = "Data\nSPM", fg = 'black', bg = 'gray', relief = 'raised', bd = BORDER, command = clickSPM)
-    button.place(height=memHeight, width=MEM_WIDTH, x = 25, y = 35)
-
+    spmLabel = tkinter.Button(canvas, text = "Data\nSPM", fg = 'black', bg = 'gray', relief = 'raised', bd = BORDER, command = clickSPM)
+    #button.place(height=memHeight, width=MEM_WIDTH, x = 25, y = 35)
+    canvas.create_window(baseX+BORDER, BORDER, window=spmLabel, height=GRID_HEIGHT, width=MEM_WIDTH, anchor="nw")

```

Comment:
Can you add a comment explaining the `anchor`?

Reply:
Didn't see how this is resolved? Once you resolve a comment, can you please reply in this dialogue window (instead of just closing it), so it is more clear to the reviewer?

Reply:
Sorry, I forgot this one. Here I use anchor="nw" to express that the data memory will be placed in the upper left corner of the cgrapannel. This way, it will be adaptive to changes in window size.